src/data/sym/basic.lean

```diff
@@ -177,7 +209,8 @@ lemma exists_eq_cons_of_succ (s : sym α n.succ) : ∃ (a : α) (s' : sym α n),
 begin
   obtain ⟨a, ha⟩ := exists_mem s,
   classical,
-  exact ⟨a, s.erase a ha, (s.cons_erase _ _).symm⟩,
+  use [a, s.erase a ha],
+  rw cons_erase,
```

Comment:
Untested but you could probably infer the second argument.

Reply:
You almost certainly can, but really I was just trying to reduce the diff to be "ok, we just changed the arguments", rather than "something went wrong with elaboration so we made the proof longer"